cfgov/jinja2/v1/_includes/molecules/full-width-text-anchor.html

```diff
@@ -0,0 +1,25 @@
+
+{# ==========================================================================
+
+   Full Width Text with Anchor
```

Comment:
Why create a new variant module for this instead of just adding it as a standard field on Full Width Text?